commitlint/plugins.ts

```diff
@@ -272,6 +272,24 @@ export abstract class Plugins {
         ];
     }
 
+    public static properRevertMessage(headerStr: string, bodyStr: string) {
+        let offence = false;
+
+        if (headerStr.match(/^[Rr]evert ".+"$/) !== null) {
+            // does msg have a body?
+            if (bodyStr !== "") {
+                let lines = bodyStr.split("\n");
+                offence =
+                    lines[0].match(/^This reverts commit [^ ]{40}\.$/) !==
+                        null && lines.length == 1;
```

Comment:
`lines.length == 1` should go before accessing `lines[0]` or this might throw an exception

Reply:
Since I it's in `if (bodyStr !== "")` block there is at least one line. The purpose of `lines.length == 1` condition is to check if there isn't more than one line because if there is, the revert message has definitely changed.

Reply:
still, placing the length check before the match is valuable because of 1. readability and 2. performance (micro-optimization I know)

Reply:
Done